src/components/Table/components/TableHeader.js

```diff
@@ -0,0 +1,34 @@
+import { useState } from 'react';
+import './TableHeader.scss';
+import { Input } from 'antd';
+import searchIcon from '../../../images/svg/searchIcon.svg';
+
+const TableHeader = ({ totalCount, handleSearch }) => {
+  const [input, setInput] = useState('');
+
+  return (
+    <div className="table-header">
+      <div className="header-content">
+        <h2 className="header-title">All Projects</h2>
+        <div className="project-count">{totalCount} total</div>
+      </div>
+      <form
+        className="search-form"
+        onSubmit={(event) => {
```

Comment:
move this on submit outside of jsx. this Whole block can be part of some function